build_config.rb

```diff
@@ -8,7 +8,7 @@
     toolchain :gcc
   end
 
-  enable_debug
+  #enable_debug
```

Comment:
@takahashim: first, there is a separate debug build. Second, it is still compiled with -g.
`enable_debug ` sets `-O0`, which is a bad idea (for the default build) IMHO.